docs/tutorials/dqn.rst

```diff
@@ -137,7 +137,7 @@ The meaning of each parameter is as follows (full description can be found at :f
 
 * ``max_epoch``: The maximum of epochs for training. The training process might be finished before reaching the ``max_epoch``;
 * ``step_per_epoch``: The number of step for updating policy network in one epoch;
-* ``collect_per_step``: The number of frames the collector would collect before the network update. For example, the code above means "collect 10 frames and do one policy network update";
+* ``step_per_collect``: The number of frames the collector would collect before the network update. For example, the code above means "collect 10 frames and do one policy network update";
```

Comment:
should be changed accordingly

Reply:
step_per_collect meaning doesn't change, but i fix meaning of step_per_epoch.